### PR TITLE
Allow undefined GroundPolylineGeometry for data that filters to a single point

### PR DESCRIPTION
**Pull request by likangning93**
Fixes https://github.com/AnalyticalGraphicsInc/cesium/issues/6702

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@likangning93, thanks for the pull request! Maintainers, we have a signed CLA from @likangning93, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Specs/Core/GroundPolylineGeometrySpec.js [hunk 1]
```diff
@@ -235,6 +235,21 @@ defineSuite([
...
+            positions : Cartesian3.fromRadiansArrayHeights([
+                startCartographic.longitude, startCartographic.latitude, 0.0,
+                startCartographic.longitude, startCartographic.latitude, 0.0
+            ]),
+            granularity : 0.0
+        });
+
+        var geometry = GroundPolylineGeometry.createGeometry(groundPolylineGeometry);
+
+        expect(geometry).not.toBeDefined();
```
Comment (hpinkos) [inline Specs/Core/GroundPolylineGeometrySpec.js]: `.not.toBeDefined()` -> `.toBeUndefined()`

File: Source/Core/GroundPolylineGeometry.js [hunk 2]
```diff
@@ -457,6 +457,11 @@ define([
         cartographics = arrayRemoveDuplicates(cartographics, Cartographic.equalsEpsilon);
         cartographicsLength = cartographics.length;
 
+        // In order to support external data that may have errors we treat this as an empty geometry.
```
Comment (hpinkos) [inline Source/Core/GroundPolylineGeometry.js]: We generally don't use inline comments unless something is truly non-obvious.  It's easy for the inline comments to get out of sync with the code when we make changes.
We honestly should probably remove most of the comments in the files you added.

**Comment by likangning93** [comment]
@hpinkos updated! I took out a few comments... there were already some that were out of sync >_<

**Comment by hpinkos** [comment]
Thanks @likangning93!  Please open another PR to remove inline comments from other files you've committed.

**Pull request merged**